core/renderers/geras/info.ts

```diff
@@ -360,7 +360,8 @@ export class RenderInfo extends BaseRenderInfo {
             row.width < prevInput.width) {
           rowNextRightEdges.set(row, prevInput.width);
         } else {
-          nextRightEdge = row.width;
+          // To avoid jagged right edges, use the maximum width.
+          nextRightEdge = Math.max(nextRightEdge, row.width);
```

Comment:
Upon looking into it further, I think my fix isn't right either.

If I revert this change, the original geras renderer sometimes renders the newline feature like this, with a weird corner on the top of the block:
![image](https://user-images.githubusercontent.com/54517768/230677575-b6cd11ed-5810-4b2c-b992-baf1bada25b6.png)

My change here fixes that... but also alters the behavior of rows that are separated by a statement input, using the widest non-statement row width so far (iterating upward through the rows):
![image](https://user-images.githubusercontent.com/54517768/230677592-c7fbd83c-8976-4936-8709-145494cd6912.png)

Note that this geras override of the method alignRowElements_() only applies if the block's inputs are inline, in which case, up until now, all consecutive inline value inputs were expected to end up in a single row, possibly separated by statement rows. The statement rows are supposed to allow the inline value rows to have different widths (but at least as wide as the statement input connectors), which I assume is the entire point of this method override. (None of the other renderers override alignRowElements_().) 

The new feature is the possibility of multiple adjacent rows of inline value inputs, or even rows containing only text labels. Without a statement row to separate these rows, I assume they ought to be the same width... and the implicit top row that's inserted for rendering corners and a margin should also be the same width, but it seems to only be as wide as the minimum width of the first explicit row, hence the weird corner I was trying to fix.

Okay, I think I've figured out the fix for real this time, I'm using the old logic for statement rows and the new logic otherwise, and it renders like this:
![image](https://user-images.githubusercontent.com/54517768/230680051-0bc7decc-e892-41c4-82ab-6a03d91b12db.png)